covid_graphs/covid_graphs/country_graph.py

```diff
@@ -131,12 +136,9 @@ def adjust_xlabel(date: datetime.date):
             plot_bgcolor="White",
         )
 
-        maximal_y = max(
-            max(curve.y_max for curve in self.curves), max(self.cropped_cumulative_active),
-        )
         # Note: We silently assume `self.cropped_cumulative_active` does not contain zeros.
         self.log_yrange = [
-            math.log10(self.cropped_cumulative_active.min()) - 0.3,
+            math.log10(max(0.1, self.cropped_cumulative_active.min())) - 0.3,
```

Comment:
I strongly suggest to use `1.0` instead of `0.1`. Otherwise the lower bound can be negative, and the graph will stretch infinitely long near zero.... I'm surprised if the displayed graph does not break actually.